### ISSUE TITLE
Syntax error messages cannot be silenced for load_html/3

### ISSUE THREAD

**Issue by wouterbeek** [SWI-Prolog/packages-sgml#4]
Syntax error messages cannot be silenced for `load_html/3`.

Reproducible example:

``` prolog
?- [library(sgml)].
?- [library(http/http_open)].
?- IRI = 'https://lists.w3.org/Archives/Public/w3c-rdfcore-wg'.
?- load_html($IRI, _, [syntax_errors(quiet)]).
SGML: Redefined entity "lt"
SGML: Redefined entity "gt"
SGML: Redefined entity "amp"
SGML: Redefined entity "apos"
SGML: Redefined entity "quot"
```

**Comment by JanWielemaker** [comment]
It comes from loading the HTML DTD in an XML context, not from the document itself.  Not sure
what to think about this.  xhtml is dead anyway ...

**Comment by triska** [comment]
I ran into the exact same issue on other sites and would greatly appreciate if this warning were also silenced. For example, I need HTML parsing in cronjobs, and currently I must use:

    swipl parse_html.pl &> /dev/null

to suppress such output.

The severe drawback: This also suppresses other output that is far more important, such as if the `swipl` executable is not found.

**Comment by JanWielemaker** [comment]
Fixed  with 46e48182ab03ac4ac8fd8bbfcd70c276bc780858
